site/packages/aa-core/smart-account-client/actions/signMessageWith6492.md

```diff
@@ -41,7 +41,7 @@ A Promise containing the signature of the message, additionally wrapped in EIP-6
 
 ## Parameters
 
-### `msg: string | Uint8Array`
+### `message: string | Uint8Array`
```

Comment:
I'm not sure the type is correct here either. It should be `SignableMessage` from viem

Reply:
yeah, should be `SignableMessage` type for `message` and another nit: return type is set to Promise<Hex> in our code, so would be nice that can be changed as well.

Reply:
yup good catch